### ISSUE TITLE
LUC carbon tax radically switches aggregated food commodities (e.g. MiscCrop, NutsSeeds) to "Tree" variants

### ISSUE THREAD

**Issue by djvdven** [JGCRI/gcam-core#308]
When running a policy with a carbon tax applies to CO2_LUC (in this case, up to 100$(2015)/tCO2 in 2050), you see in some aggregated crop commodities like MiscCrop and NutsSeeds a radical change towards producing this crop commodity with the "Tree" variant (MiscCropTree). While this makes sense as the "Tree" variant has higher carbon contents, it implies a radical change of dietary preferences as basically MiscCropTree are mainly different crops than MiscCrop, just they are aggregated to the same MiscCrop final demand. Since the model by default assumes cultural food preferences are persistent in food demand (e.g. using logits of zero in the food demand tree), this model behavior of switching to "Tree" is not consistent with that default assumption.

**Comment by pkyle** [comment]
I'm impressed that the model is returning this result! It's not something I had noticed in our testing of that specific disaggregation, which was made public within the GCAM 6.0 release, in June 2022.
Allowing this carbon -> price driven switch from field crops to tree crops seems realistic, though there is a question of the degree, and in context of the specific substitution options available within the nests where it's observed. Some tree crops have carbon contents that are similar to mature forests; at the limit, Brazil nut trees basically require healthy, mature forest canopies below. Cashew orchards similarly have higher average carbon densities per unit land area than many temperate woodlots. But the heart of the question is the degree of substitutability between field and tree crops within GCAM commodities; such competition is seen within Fruits, NutsSeeds, OilCrop, and MiscCrop.
Regarding the fact that inter-crop substitution is still turned off by default in GCAM, this is a known issue though the appropriate resolution isn't simple given that the sorts of commodity prices that we see in certain scenarios are well out of sample for the current literature on elasticities. The revised food demand model (merged in 2022, prior to GCAM 6.0) is focused on total caloric inputs by staples and non-staples, but doesn't address inter-crop competition within any given GCAM commodit class.
